src/main/lrsql/input/actor.clj

```diff
@@ -61,6 +61,18 @@
    :actor-ifi    actor-ifi
    :actor-type   actor-type})
 
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Actor Delete
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(s/fdef delete-actor-input
```

Comment:
Weird spacing with the blank lines on lines 64 and 65, and the lack thereof after line 68?

Reply:
adjusted